src/Microsoft.Azure.WebJobs.Host/Tables/StringToStorageTableConverter.cs

```diff
@@ -1,38 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using Microsoft.Azure.WebJobs.Host.Converters;
-using Microsoft.Azure.WebJobs.Host.Storage.Table;
-
-namespace Microsoft.Azure.WebJobs.Host.Tables
-{
-    internal class StringToStorageTableConverter : IConverter<string, IStorageTable>
```

Comment:
The majority of the converter logic is unchanged (this brings up another issue: https://github.com/Azure/azure-webjobs-sdk/issues/974) .  These are some top-level converters, also to IStorageTable (which is the internal test interface).  I'll bang at it more before the final commit.